Sources/BraintreePayPal/BTPayPalRequest_Internal.h

```diff
@@ -11,11 +11,18 @@ NS_ASSUME_NONNULL_BEGIN
 extern NSString *const BTPayPalCallbackURLHostAndPath;
 extern NSString *const BTPayPalCallbackURLScheme;
 
+typedef NS_ENUM(NSUInteger, BTPayPalPaymentType) {
+    BTPayPalPaymentTypeCheckout,
+    BTPayPalPaymentTypeBillingAgreement
```

Comment:
Maybe we should rename this to `BTPayPalPaymentTypeVault` to align with the request type name

Reply:
Good call! Updated.